prometheus_api_client/prometheus_connect.py

```diff
@@ -196,6 +196,9 @@ def get_metric_range_data(
         if not (isinstance(start_time, datetime) and isinstance(end_time, datetime)):
             raise TypeError("start_time and end_time can only be of type datetime.datetime")
 
+        if end_time < start_time:
```

Comment:
Can you also add this check in all other range query functions which utilize the start_time and end_time parameters?

Reply:
Suree